se2_planning_ros/src/GridMapLazyStateValidatorRos.cpp

```diff
@@ -0,0 +1,92 @@
+/*
+ * GridMapLazyStateValidatorRos.cpp
+ *
+ *  Created on: Jul 20, 2020
+ *      Author: meyerc
+ */
+
+#include "se2_planning_ros/GridMapLazyStateValidatorRos.hpp"
+
+#include <thread>
+
+namespace se2_planning {
+
+GridMapLazyStateValidatorRos::GridMapLazyStateValidatorRos(ros::NodeHandlePtr nh) : BASE(), nh_(nh), newMapAvailable_(false) {}
```

Comment:
this merely listens to ros topics and updates the map, correct?

Reply:
Yes.